src/login/Login.jsx

```diff
@@ -25,21 +25,26 @@ export default function Login() {
             },
             body: JSON.stringify(payload)
         };
-
         fetch("http://127.0.0.1:5000/login", requestLogin)
             .then(async response => {
                 let data = await response.json();
                 if (response.status === 200) {
```

Comment:
Should be a constant that can be directly imported everywhere.

Reply:
make sense. Changed it now to constant 👍

Reply:
We can move it to a config file - you can check this out for reference - https://github.com/thoughtworks/epirust/blob/master/web/react-spa/src/config.js

see how the API_HOST is abstracted out. 

After a period of time, you may not realize the constant is in this file. This is more of config information anyway, would be nice to abstract it completely. The link to refer should help

Reply:
Yeah.... I did this coz you always prefer/advise to avoid unnecessary abstraction and making separate files just for trivial thing, as you said "wait until it gets dirty then abstract them".
I'm just following your advise and though this is only one constant you might not want to see it abstracte out to a separate files. 
I myself prefer to do the "cleaner" way (meaning if I know we are going to use it often and will likely grow to a bggger size I always like to keep it separate from the very first line I encounter). 
Please let me know if this preference of yours (wait tiil it gets "dirty") no longer valid so we can do the "cleaner" way from this point onwards. But it needs to be consistent to save me time from keep changing to inconsistent style preferences.

Reply:
Updatte. 've just pushed another commit with BASE_API now placed in a config.js file